buffer/src/main/java/io/netty/buffer/AbstractByteBuf.java

```diff
@@ -1293,7 +1293,7 @@ public int forEachByte(int index, int length, ByteProcessor processor) {
         }
     }
 
-    private int forEachByteAsc0(int start, int end, ByteProcessor processor) throws Exception {
+    int forEachByteAsc0(int start, int end, ByteProcessor processor) throws Exception {
```

Comment:
```suggestion
    protected int forEachByteAsc0(int start, int end, ByteProcessor processor) throws Exception {
```

Reply:
I thought it may be better to start with more restricted level. We can open it up later if we really want to.